studio/lib/constants/infrastructure.ts

```diff
@@ -132,3 +132,5 @@ export const PASSWORD_STRENGTH_PERCENTAGE = {
 }
 
 export const DEFAULT_FREE_PROJECTS_LIMIT = 2
+
+export const STORAGE_FILE_SIZE_LIMIT_MAX = 5_368_709_120
```

Comment:
likewise from the API could we display this as a product (x * y * z) so its easier to see how many MB